File: app/src/main/java/org/mifos/mobilebanking/ui/activities/HelpActivity.java
package org.mifos.mobilebanking.ui.activities;

import android.os.Bundle;

import org.mifos.mobilebanking.R;
import org.mifos.mobilebanking.ui.activities.base.BaseActivity;
import org.mifos.mobilebanking.ui.fragments.HelpFragment;

/**
 * @author Rajan Maurya
 * On 11/03/19.
 */
public class HelpActivity extends BaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_container);
        setToolbarTitle(getString(R.string.help));
        showBackButton();
        replaceFragment(HelpFragment.newInstance(), false, R.id.container);
    }
}

File: app/src/main/java/org/mifos/mobile/ui/help/HelpActivity.kt
package org.mifos.mobile.ui.help

import android.os.Bundle
import android.view.View
import org.mifos.mobile.R
import org.mifos.mobile.databinding.ActivityContainerBinding
import org.mifos.mobile.ui.activities.base.BaseActivity

/**
 * @author Rajan Maurya
 * On 11/03/19.
 */
class HelpActivity : BaseActivity() {
    private lateinit var binding: ActivityContainerBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityContainerBinding.inflate(layoutInflater)
        setContentView(binding.root)
        toolbar?.visibility = View.GONE
        replaceFragment(HelpFragment.newInstance(), false, R.id.container)
    }
}
